app/controllers/internal_api/v1/pillars_controller.rb

```diff
@@ -112,6 +112,7 @@ def azure_cloud_contents
   def openstack_cloud_contents
     {
       cloud: {
+        provider: "openstack",
```

Comment:
Instead of hardcoded value I would use `Pillar.value(pillar: :cloud_provider)`.

Reply:
Given that the method is called `openstack_cloud_contents` I don't think it's much of an issue, no?

Reply:
Gotcha. Just saw that `cloud_provider_contents` calls `openstack_cloud_contents` when it's `cloud_provider` is `openstack`. Then it's fine to be like this. 😅

Reply:
Yes, this should stay hardcoded.